pkg/proxy/ipvs/netlink_linux.go

```diff
@@ -182,6 +182,9 @@ func (h *netlinkHandle) GetLocalAddresses(dev, filterDev string) (sets.String, e
 		if route.LinkIndex == filterLinkIndex {
 			continue
 		}
+		if route.Dst.IP.IsLoopback() {
+			continue
```

Comment:
do you want to log something?
like "loopback addresses no supported for nodeports on ipvs mode"
?

Reply:
No, not really. There is no good place to log something since this is not a configuration.

Perhaps for nodePortAddresses settings, but nah... we set nodePortAddresses to loopback to avoid exposing nodePorts on port scans, and that is still a valid use-case I would say. No need for logging.

Reply:
IMHO semantically `GetLocalAddresses()` is correct returning the loopback .

shouldn't be better to filter the loopback in the proxier.go code ? it is only used in 2 places.
You can do it here
https://github.com/kubernetes/kubernetes/blob/57459ebd3e56eb0747e4697422e8ac24f7d113c6/pkg/proxy/ipvs/proxier.go#L304-L314

the other place that is being used is 
https://github.com/kubernetes/kubernetes/blob/57459ebd3e56eb0747e4697422e8ac24f7d113c6/pkg/proxy/ipvs/proxier.go#L317-L320
not familiar with Bind but seems legit to bind to loopbacks

Reply:
Good point :+1:  I will fix that (and squash the commits at the same time).

Reply:
Update pushed. Thanks for spotting this :+1:  It exposed a mistake that I updated only `netlink_linux.go`. Now I had to update the unit-tests which I was suprised that I didn't have to do before.

Reply:
I have also changed the comment to "Disable loopback:nodeport for proxy-mode=ipvs" which is better than "localhost:nodeport" which is not exactly the same.